docs/source/development-environment.rst

```diff
@@ -0,0 +1,141 @@
+Zipline Development Guidelines
+==============================
+
+Environment Setup
+-----------------
+
+First, you'll need to clone zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:quantopian/zipline.git
```

Comment:
Is this for quantopian devs or community devs? Those without commit rights to the quantopian repo will need to clone their own forks.